packages/bot-skeleton/src/services/tradeEngine/trade/Total.js

```diff
@@ -23,10 +23,10 @@ export default Engine =>
             this.sessionRuns = 0;
             this.sessionProfit = 0;
 
-            globalObserver.register('summary.clear', this.clearSummary.bind(this));
+            globalObserver.register('state.clear', this.clearState.bind(this));
```

Comment:
Could we call this event `statistics.clear`? `state` is a bit too generic imo.

Reply:
sure.